bin/yunoprompt

```diff
@@ -51,17 +54,35 @@ if [[ -f /etc/yunohost/installed ]]
 then
     echo "$LOGO_AND_FINGERPRINTS" > /etc/issue
 else
-    sleep 5
     chvt 2
+
+    # Formatting
+    [[ -n "$local_ip" ]] && local_ip=$(echo -e "https://$local_ip/") || local_ip="(no ip detected?)"
+
     echo "$LOGO_AND_FINGERPRINTS"
-    echo -e "\e[m Post-installation \e[0m"
-    echo "Congratulations! YunoHost has been successfully installed.\nTwo more steps are required to activate the services of your server."
-    read -p "Proceed to post-installation? (y/n)\nAlternatively, you can proceed the post-installation on https://${ip}" -n 1
+    cat << EOF
+===============================================================================
+You should now proceed with Yunohost post-installation. This is where you will
+be asked for :
+  - the main domain of your server ;
+  - the administration password.
+
+You can perform this step :
+  - from your web browser, by accessing : ${local_ip}
+  - or in this terminal by answering 'yes' to the following question
+
+If this is your first time with YunoHost, it is strongly recommended to take
+time to read the administator documentation and in particular the sections
+'Finalizing your setup' and 'Getting to know YunoHost'. It is available at
+the following URL : https://yunohost.org/admindoc
+===============================================================================
+EOF
```

Comment:
Isn't it too long to be displayed on screen ?

Reply:
We could make a smaller logo on 3 lines to get more lines to explain.

Reply:
Idk, tested it in a virtualbox with the default "small" screen and it looked fine

Reply:
ok cool